cranelift/frontend/src/switch.rs

```diff
@@ -241,6 +264,13 @@ impl Switch {
         cases_and_jt_blocks: Vec<(EntryIndex, Block, Vec<Block>)>,
     ) {
         for (first_index, jt_block, blocks) in cases_and_jt_blocks.into_iter().rev() {
+            // There are currently no 128bit systems supported by rustc, but once we do ensure that
+            // we don't silently ignore a part of the jump table for 128bit integers on 128bit systems.
+            assert!(
```

Comment:
Can we just assert that the values are `<= i64::MAX` (or use `i64::try_from` below) instead? It seems fine to me to build on a 128-bit system and generate 64-bit-sized IR as long as the statically-known values are in range (unless I'm missing some other issue!).

Reply:
Changed this to assert that `blocks.len()` fits in an `u64`.